apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/sink/direct/FissClaimRdaSink.java

```diff
@@ -38,4 +39,17 @@ public long getSequenceNumberForObject(FissClaimChange object) {
     change.getClaim().setApiSource(apiVersion);
     return change;
   }
+
+  @Override
+  RdaApiClaimMessageMetaData createMetaData(RdaChange<PreAdjFissClaim> change) {
```

Comment:
`@Override` should have the `/* {@inheritdoc} */` javadoc.

Reply:
`inheritDoc ` is only relevant when adding adding additional comments.  Adding a comment containing only that would not add any value.  See [javadoc documentation](https://docs.oracle.com/javase/8/docs/technotes/tools/windows/javadoc.html) for details.  I have added a comment in the base class method that will be inherited automatically.